src/managers/input/InputManager.cpp

```diff
@@ -1381,6 +1381,43 @@ void CInputManager::refocus() {
     mouseMoveUnified(0, true);
 }
 
+void CInputManager::refocusLastWindow(CMonitor* pMonitor) {
+    if (!pMonitor) {
+        refocus();
+        return;
+    }
+
+    Vector2D               surfaceCoords;
+    PHLLS                  pFoundLayerSurface;
+    SP<CWLSurfaceResource> foundSurface = nullptr;
+
+    g_pInputManager->releaseAllMouseButtons();
+    g_pCompositor->m_pLastFocus.reset();
```

Comment:
This is just refactored from LayerSurface, but it seems odd that focusWindow() and refocus() don't handle this.

Reply:
wdym dont handle this?

Reply:
seems like focusing a surface should already release the mouse buttons if its different. Not sure what the last focus clear is doing.

Reply:
me neither, prolly can be removed. Reset only removes the pointer, does no checks or unfocuses.